runners/apex/src/main/java/org/apache/beam/runners/apex/translation/operators/ApexParDoOperator.java

```diff
@@ -83,6 +105,11 @@
   @Bind(JavaSerializer.class)
   private final List<PCollectionView<?>> sideInputs;
 
+  private StateInternalsProxy<?> currentKeyStateInternals;
+  // TODO: fault tolerant timer internals
```

Comment:
What are the implications of currently having non-fault tolerant ones? And what faults are these not tolerant to?